src/exporthtml.ts

```diff
@@ -459,6 +471,8 @@ function formatContent(content: string, context: discord.NewsChannel | discord.T
                 return `<span class="pre pre--inline">${joined.substring(0, 42)}</span>`;
             }
         })
+        .replace(/<a:(.+):(\d+)>/g, (f, b, o) => { return `<img src="https://cdn.discordapp.com/emojis/${o}.gif?size=96&quality=lossless" width="${(/^<a:(.+):(\d+)>$/).test(text) ? '48px' : '22px'}" height="${(/^<a:(.+):(\d+)>$/).test(text) ? '48px' : '22px'}">`; })
```

Comment:
Change to something like
```js
.replace(/\&lt\;a:(.+?):(\d+?)\&gt\;/g, (_content, _name, id) => `<img src="https://cdn.discordapp.com/emojis/${id}.gif?size=96" class="emoji ${emojiClass}">`)
```
because you need to escape the `<` and `>` since this is after running the purify function.

Define emojiClass as the following in the top of the function:
```js
    const emojiClass = // /^(<(a?):(.+?):(\d+?)>([ \t]+?)?){0,27}$/.test(content) < something is wrong with this regex - I would prefer to use this
        Array.from(content.matchAll(/(<(a?):(.+?):(\d+?)>([ \t]+?)?)/g)).length <= 27
        ? `emoji--large` : `emoji--small`;
```